samples/arm-none-eabi/reported.by

```diff
@@ -0,0 +1,310 @@
+reporter_name="Jasmin Jessich"
+reporter_url="http://jasmin.anw.at/"
+reporter_comment="NOTE: Please read samples/arm-none-eabi/reported.by prior to any built!"
+
+#
+# This configuration will build an arm-none-eabi compiler to be used for
+# embedded development. The original build script can be found here:
+#    https://launchpad.net/gcc-arm-embedded
+# To see the original build script, you have to download the installer:
+#    https://launchpad.net/gcc-arm-embedded/5.0/5-2015-q4-major
+# 
+# I tried to mimic the original configure options and environment settings with
+# crosstool-ng (CT_NG) as close as possible. I really analysed every option
+# which was not provided by CT-NG or which was added. The description in this
+# file explains each diversion from the original.
+# 
+# NOTE:   newlib-nano is currently not built!
+# 
+# The original uses the git/SVN versions of binutils-gdb, gcc and newlib. CT-NG
+# doesn't currently support using a git or SVN repository for the source
+# location. In fact this is not really necessary, because the official packages
+# contain most of the required code already.
+# Only armv8-m is not available in the official packages. This core is very new
+# and even not available on GCC trunk, so I guess we can wait for the next GCC
+# release to add it.
+#
+# If someone really want to experiment with the mentioned core variant, feel
+# free to use them as custom location in CT-NG.
```

Comment:
> Confirmed: built fine for me, too. Thanks again!

Fine!

> As a side note, I wonder why ARM GCC does not allow similar precise multilib specification
> for t-aprofile ...

His words:
"Note that arm/t-aprofile is a stand-alone make file fragment to be used only with itself.  We
do not specifically use the TM_MULTILIB_OPTION framework because this shorthand is
more pragmatic."
I guess the developer (Ramana Radhakrishnan) didn't saw it as a requirement, because you normally build a compiler not very frequently. Then it doesn't matter if you compile multilib for only one or more targets, because the build time will be not increased very much.
Or he simply was to lazy ;)